tests/model/test_model.py

```diff
@@ -3346,6 +3347,46 @@ def test__handle_subscription_event_subscribers_one_user_multiple_streams(
             new_subscribers = model.stream_dict[stream_id]["subscribers"]
             assert new_subscribers == expected_subscribers
 
+    @pytest.mark.parametrize(
+        "person, changed_details",
+        [
+            (
+                {"user_id": 11, "full_name": "New Full Name"},
+                "full_name",
+            ),
+            ({"user_id": 11, "timezone": "New Timezone"}, "timezone"),
+            ({"user_id": 11, "is_billing_admin": False}, "is_billing_admin"),
+            ({"user_id": 11, "role": 10}, "role"),
+            (
+                {"user_id": 11, "avatar_url": "new_avatar_url", "avatar_version": 21},
+                "avatar_url",
+            ),
+            ({"user_id": 11, "new_email": "new_display@email.com"}, "email"),
+            (
+                {"user_id": 11, "delivery_email": "new_delivery@email.com"},
+                "delivery_email",
+            ),
+        ],
+        ids=[
```

Comment:
There a lot of test cases & ids, so it's simpler to keep them inline - then later we wouldn't have to check where in each list to add/remove/edit them, or to move them around.